build/steps-init-codecheck.yml

```diff
@@ -0,0 +1,9 @@
+steps:
```

Comment:
Given that this pipeline is a lot simpler than the others, it may make sense to just directly include the pre-reqs in the main YAML.

Reply:
Merged `steps-init-codecheck.yml` to `steps-codecheck.yml`.